app/controllers/admin/themes_controller.rb

```diff
@@ -4,6 +4,14 @@
 
 class Admin::ThemesController < Admin::AdminController
   MAX_REMOTE_LENGTH = 10_000
+  ALLOW_EDIT_SYSTEM_FIELDS = %w[
+    child_theme_ids
+    color_scheme_id
+    default
+    locale
+    translations
+    user_selectable
+  ]
```

Comment:
We should protect against changing these fields at the model level in a `before_save` callback to prevent another code path from accidentally updating these fields.